metaflow/mflog/save_logs.py

```diff
@@ -8,9 +8,12 @@
 from metaflow.util import Path
 from . import TASK_LOG_SOURCE
 
+from metaflow.tracing import cli_entrypoint
+
 SMALL_FILE_LIMIT = 1024 * 1024
 
 
+@cli_entrypoint("save_logs")
```

Comment:
should there be a prefix for the entrypoint or what is the naming convention for these?

Reply:
For now there's no naming convention. This is basically giving a name to the related trace so it could be used for search/filtering